docs/contributing/developing.rst

```diff
@@ -183,17 +183,16 @@ Browser        Device/OS      Version(s)
 Mobile Safari  iOS Phone      Last 2
 Mobile Safari  iOS Tablet     Last 2
 Chrome         Android        Last 2
-IE             Desktop        11
 Chrome         Desktop        Last 2
 MS Edge        Desktop        Last 2
 Firefox        Desktop        Latest
 Firefox ESR    Desktop        Latest
 Safari         macOS          Last 2
 =============  =============  =============
 
-We aim for Wagtail to work in those environments. Our development standards ensure that the site is usable on other browsers **and will work on future browsers**. To test on IE, install virtual machines `made available by Microsoft <https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/>`_.
+We aim for Wagtail to work in those environments. Our development standards ensure that the site is usable on other browsers **and will work on future browsers**.
 
-IE 11 is gradually falling out of use, and specific features are unsupported in this browser:
+IE 11 support has officially dropped in 2.15 as it is gradually falling out of use, many specific features are completely unsupported in this browser:
```

Comment:
has officially dropped -> has been officially dropped

I'd be inclined to remove the list of things that don't work - it creates an expectation that things _not_ on the list ought to still work in IE11, and we're no longer promising that (and we're likely to break a lot more once we start removing compatibility shims). Or at least, we should say "features known not to work include..." - saying "unsupported" is understating things.

Reply:
Done, thanks.